<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>431</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>431</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>271</width>
    <height>431</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_20">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_19">
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_18">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_16">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <widget class="QLabel" name="label_login_gift">
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Login</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="1" column="0">
                <layout class="QGridLayout" name="gridLayout_13">
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="3">
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_9">
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_6">
                     <item row="0" column="0">
                      <spacer name="horizontalSpacer_3">
                       <property name="font">
                        <font>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item row="0" column="2">
                      <spacer name="horizontalSpacer_4">
                       <property name="font">
                        <font>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item row="0" column="1">
                      <layout class="QGridLayout" name="gridLayout_5">
                       <item row="1" column="0">
                        <layout class="QGridLayout" name="gridLayout_4">
                         <item row="0" column="1">
                          <widget class="QLineEdit" name="lineEdit_lgn_password">
                           <property name="font">
                            <font>
                             <bold>true</bold>
                            </font>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="0">
                          <widget class="QLabel" name="label_2">
                           <property name="font">
                            <font>
                             <bold>true</bold>
                            </font>
                           </property>
                           <property name="text">
                            <string>Password:  </string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item row="0" column="0">
                        <layout class="QGridLayout" name="gridLayout_3">
                         <item row="0" column="1">
                          <widget class="QLineEdit" name="lineEdit_lgn_username">
                           <property name="font">
                            <font>
                             <bold>true</bold>
                            </font>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="0">
                          <widget class="QLabel" name="label">
                           <property name="font">
                            <font>
                             <bold>true</bold>
                            </font>
                           </property>
                           <property name="text">
                            <string>Username:</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="0">
                    <layout class="QGridLayout" name="gridLayout_8">
                     <item row="0" column="2">
                      <spacer name="horizontalSpacer_2">
                       <property name="font">
                        <font>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>13</width>
                         <height>17</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item row="0" column="1">
                      <layout class="QGridLayout" name="gridLayout_7">
                       <item row="0" column="0">
                        <layout class="QGridLayout" name="gridLayout_2">
                         <item row="0" column="0">
                          <widget class="QLabel" name="label_3">
                           <property name="font">
                            <font>
                             <bold>true</bold>
                            </font>
                           </property>
                           <property name="text">
                            <string>Login</string>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="1">
                          <widget class="QPushButton" name="pushButton_login">
                           <property name="minimumSize">
                            <size>
                             <width>50</width>
                             <height>50</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>50</width>
                             <height>50</height>
                            </size>
                           </property>
                           <property name="font">
                            <font>
                             <bold>true</bold>
                            </font>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">#pushButton_login{
background-color: transparent;
border-image: url(:enter_close.png);
background: none;
border: none;
background-repeat: none;
}
#pushButton_login:pressed
{
   border-image: url(:enter_open.png);
}
</string>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item row="0" column="1">
                        <layout class="QGridLayout" name="gridLayout">
                         <item row="0" column="1">
                          <widget class="QPushButton" name="pushButton_singin">
                           <property name="minimumSize">
                            <size>
                             <width>50</width>
                             <height>50</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>50</width>
                             <height>50</height>
                            </size>
                           </property>
                           <property name="font">
                            <font>
                             <bold>true</bold>
                            </font>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">#pushButton_singin{
background-color: transparent;
border-image: url(:singin_close.png);
background: none;
border: none;
background-repeat: none;
}
#pushButton_singin:pressed
{
   border-image: url(:singin_open.png);
}
</string>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="0">
                          <widget class="QLabel" name="label_4">
                           <property name="font">
                            <font>
                             <bold>true</bold>
                            </font>
                           </property>
                           <property name="text">
                            <string>Sing in</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                     <item row="0" column="0">
                      <spacer name="horizontalSpacer">
                       <property name="font">
                        <font>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>13</width>
                         <height>17</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="4">
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Calculator</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="title">
               <string>Admin Panel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButton_adm">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#pushButton_adm{
background-color: transparent;
border-image: url(:admin_one.png);
background: none;
border: none;
background-repeat: none;
}
#pushButton_adm:pressed
{
   border-image: url(:admin_two.png);
}
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="pushButton_calculator">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#pushButton_calculator{
background-color: transparent;
border-image: url(:calculator_close.png);
background: none;
border: none;
background-repeat: none;
}
#pushButton_calculator:pressed
{
   border-image: url(:calculator_open.png);
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="0" column="0">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_lgn_status">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
